mapreader/classify/datasets.py

```diff
@@ -122,23 +176,35 @@ def __getitem__(self, idx: Union[int, torch.Tensor]) -> Tuple[torch.Tensor, Any]
         Returns
         -------
         tuple
```

Comment:
might be helpful to add what the tuple contains here - like the type hinting above `Tuple[torch.Tensor, Any]`.. that way, it'll be included when generating the reference docs for the web. (This applies to any Returns sections where applicable.)

Reply:
yes agree - have updated this and the one in PatchContextDataset